.github/workflows/ci.yml

```
@@ -134,6 +134,16 @@ jobs:
           working-directory: tests
           coverage-summary-path: coverage/coverage-summary.json
 
+      - name: Test request-coderabbit-full-review action
+        # Disabled: a live run posts a @coderabbitai comment that triggers a
+        # real CodeRabbit full review on this repo's PRs. Script logic is
+        # covered by bats-tests/shell/request-coderabbit-full-review.bats.
+        if: false
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)